reconstruct/reconstruct.go

```diff
@@ -69,6 +72,25 @@ func LikelihoodsToBase(likelihoods []float64, nonBiasBaseThreshold float64, bias
 	return answer
 }
 
+// LikelihoodsToPbase converts and normalises a slice of probabilities for one position of a sequence. That position refers to a specific base.
+func LikelihoodsToPBase(likelihoods []float64, nonBiasBaseThreshold float64, biasBase dna.Base, highestProbThreshold float64) pDna.Float32Base {
+	if len(likelihoods) < 4 {
+		log.Fatalf("Error: Expected four bases, received less.")
+	}
+
+	var total float64 = 0
+	for _, v := range likelihoods {
+		total += v
+	}
+	var answer pDna.Float32Base
+	answer.A = float32(likelihoods[0] / total)
```

Comment:
Would you want a total==0 check? Maybe something like this?
```
	if total == 0 {
		return pDna.Float32Base{
			A: 0,
			C: 0,
			G: 0,
			T: 0,
		}
	} else {
		return pDna.Float32Base{
			A: float32(likelihoods[0] / total),
			C: float32(likelihoods[1] / total),
			G: float32(likelihoods[2] / total),
			T: float32(likelihoods[3] / total),
		}
	}
```

Reply:
Thanks for catching, I've added this in.